consensus/pacemaker/debug.go

```diff
@@ -0,0 +1,30 @@
+package pacemaker
+
+import typesCons "github.com/pokt-network/pocket/consensus/types"
+
+var _ PacemakerDebug = &pacemaker{}
```

Comment:
Lmk if I'm not correct about this. Maybe it's both?


```suggestion
var _ PacemakerDebug = &pacemakerDebug{}
```

Reply:
It is pacemaker because the functions exposed by this is implemented by the pacemaker. This is a separate interface only to make the usage clear. This is same with what we did with `ConsensusDebugModule` 
```
var (
	_ modules.ConsensusModule = &consensusModule{}
	_ ConsensusDebugModule    = &consensusModule{}
)
```